_pages/resources.md

```diff
@@ -11,12 +11,12 @@ Here are some links to our resources and related sites:
 
 * [**pyOpenSci Github Organization**](https://github.com/pyOpenSci): Contains our [software-review](https://github.com/pyOpenSci/software-review) repo that hosts the peer-review process. Also contains our guidebook, cookiecutter, and governance repos (see below).
 
-* [**pyOpenSci Discourse Forum**](https://pyopensci.discourse.group/): Our community forum for discussing all things pyOpenSci. Have feedback or ideas about the review process? Need help finishing or maintaining your Python package? Have a question about how to use one of pyOpenSci’s packages? Our forum is the best place for these types of discussion. 
+* [**pyOpenSci Discourse Forum**](https://pyopensci.discourse.group/): Our community forum for discussing all things pyOpenSci. Have feedback or ideas about the review process? Need help finishing or maintaining your Python package? Have a question about how to use one of pyOpenSci’s packages? Our forum is the best place for these types of discussion.
 
-* [**pyOpenSci Guidebook**](https://www.pyopensci.org/contributing-guide/intro): Contains information for package authors and reviewers, including our [package guide](https://www.pyopensci.org/dev_guide/packaging/packaging_guide.html), our [peer review process](https://www.pyopensci.org/dev_guide/peer_review/peer_review_proc.html), and tips for [package maintenance](https://www.pyopensci.org/dev_guide/maintenance/maintenance_intro.html). If you are interested in submitting a package to pyOpenSci, start with our [author guide](https://www.pyopensci.org/dev_guide/peer_review/author_guide.html) and the links within it.
+* [**pyOpenSci Guidebook**](https://www.pyopensci.org/contributing-guide/intro): Contains information for package authors and reviewers, including our [package guide](https://www.pyopensci.org/contributing-guide/authoring/index.html#packaging-guide), our [peer review process](https://www.pyopensci.org/contributing-guide/intro.html), and tips for [package maintenance](https://www.pyopensci.org/dev_guide/maintenance/maintenance_intro.html). If you are interested in submitting a package to pyOpenSci, start with our [author guide](https://www.pyopensci.org/contributing-guide/authoring/index.html) and the links within it.
```

Comment:
All the `dev_guide` URLs were also broken as they don't exist anymore either. Some were easy to fixup, but there is no longer a maintenance section on the Contributing Guide, so I'm not sure what you want to do with 

```markdon
and tips for [package maintenance](https://www.pyopensci.org/dev_guide/maintenance/maintenance_intro.html)
```

which is currently broken.

Reply:
thank you so much for this  @matthewfeickert   !!  i'll have a look at this !!!

Reply:
Sure. Let me know if you have any questions and I'm obviously happy to revise this as much as needed or scrap this PR in favor of something larger.

Reply:
ok @matthewfeickert  this section was reorganized last year BUT... i think the link for maintainence can go here now - https://www.pyopensci.org/contributing-guide/open-source-software-peer-review/policies-and-guidelines.html#after-acceptance-package-ownership-and-maintenance ... it was a while back but it looks like we integrated that content into the policies and guidelines